### PR TITLE
DT-1255 Remove interfaces from resource controllers

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/api/resource/AgencyResource.java [hunk 1]
```diff
@@ -21,155 +21,195 @@
...
-    ResponseEntity<List<Agency>> getAgencies(@ApiParam(value = "Requested offset of first record in returned collection of agency records.", defaultValue = "0") @RequestHeader(value = "Page-Offset", defaultValue = "0", required = false) Long pageOffset,
-                                    @ApiParam(value = "Requested limit to number of agency records returned.", defaultValue = "10") @RequestHeader(value = "Page-Limit", defaultValue = "10", required = false) Long pageLimit);
+    @ApiOperation(value = "List of active agencies.", notes = "List of active agencies.", nickname = "getAgencies")
+    @GetMapping
+    public ResponseEntity<List<Agency>> getAgencies(@RequestHeader(value = "Page-Offset", defaultValue = "0", required = false) @ApiParam(value = "Requested offset of first record in returned collection of agency records.", defaultValue = "0") final Long pageOffset, @RequestHeader(value = "Page-Limit", defaultValue = "10", required = false) @ApiParam(value = "Requested limit to number of agency records returned.", defaultValue = "10") final Long pageLimit) {
```
Comment (mikehalmamoj) [inline src/main/java/uk/gov/justice/hmpps/prison/api/resource/AgencyResource.java:56]: I'd be tempted to split these out onto individual rows for readability.

File: src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/personalcareneeds_offender_validation.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "status": 400,
-  "userMessage": "getPersonalCareNeeds.offenderNo: offenderNo: must not be empty",
-  "developerMessage": "getPersonalCareNeeds.offenderNo: offenderNo: must not be empty"
+  "userMessage": "getPersonalCareNeeds.offenderNos: offenderNo: must not be empty",
```
Comment (andymarke) [inline src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/personalcareneeds_offender_validation.json:3]: How come this has changed?

**Review by andymarke — APPROVED** [review]
LGTM

File: src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/personalcareneeds_offender_validation.json (hunk 2 above)
Comment (Mjwillis) [reply src/test/resources/uk/gov/justice/hmpps/prison/api/resource/impl/personalcareneeds_offender_validation.json:3]: Interface was different to implementation in variable name

**Pull request merged**